src/parallel_aggregate.workflows.yaml

```diff
@@ -31,7 +34,7 @@ main:
               - tamingoftheshrew
               - loveslabourslost
     - runQueries:
-        parallel:  # numWords is shared so it can be written within the parallel loop
+        parallel:  # `numWords` is shared so it can be written within the parallel loop
```

Comment:
not. it is possible ` charcters might have adverse affects with our parser, recommend using ' instead (since ` can be used to denote multi-line text but ' is not)

Reply:
Sure, done